components/camera/replaypcd/replaypcd.go

```diff
@@ -50,10 +50,12 @@ func init() {
 
 // Config describes how to configure the replay camera component.
 type Config struct {
-	Source    string       `json:"source,omitempty"`
-	RobotID   string       `json:"robot_id,omitempty"`
-	Interval  TimeInterval `json:"time_interval,omitempty"`
-	BatchSize *uint64      `json:"batch_size,omitempty"`
+	Source         string       `json:"source,omitempty"`
+	RobotID        string       `json:"robot_id,omitempty"`
+	LocationID     string       `json:"location_id,omitempty"`
+	OrganizationID string       `json:"organization_id,omitempty"`
```

Comment:
Would you mind updating the SLAM QA plan (https://docs.google.com/document/d/1QwT1IJohVk61FmADlZ4KHBxbfz4zSPm7aogKlZZ1i8o/edit) and sample replay camera config (https://github.com/viamrobotics/slam-configs/blob/main/viam-cartographer/2d/replay.json) based on these changes?